rocAL/rocAL/source/meta_data/bounding_box_graph.cpp

```diff
@@ -31,14 +31,14 @@ void BoundingBoxGraph::process(pMetaDataBatch meta_data)
 
 //update_meta_data is not required since the bbox are normalized in the very beggining -> removed the call in master graph also except for MaskRCNN
 
-inline double ssd_BBoxIntersectionOverUnion(const BoundingBoxCord &box1, const double &box1_area, const BoundingBoxCord &box2)
+inline double ssd_BBoxIntersectionOverUnion(const BoundingBoxCord &box1, const double &box1_area, const BoundingBoxCordInfo &box2)
```

Comment:
The processing in this function needs to be done in float
So typecast the box1 to float and all the others should be in float only